apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -1,5 +1,6 @@
 import { createContext, useEffect, useMemo, useCallback, useContext, useState } from 'react';
-import { useFormContext, useFieldArray } from 'react-hook-form';
+import { FormProvider, useForm, useFieldArray } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
```

Comment:
Should we move all validations in the FE to use Zod? What do you suggest?

Reply:
we do use `zod` for the notification template schema and `react-hook-form` validations... 
I personally haven't used it before, but I heard good options about it... we can give it a shot! :)